### ISSUE TITLE
Contribute: tweak to validation message

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#4144]
When a user gives a Dataset Title that already exists, the message is 

"That URL is already in use."

While this is perfectly true, it may confuse a contributor, especially if they are given a URL resource.  I suggest we change the error message to

"That title is already in use.  Please edit your title or the URL just below it."

![image](https://cloud.githubusercontent.com/assets/1654485/13495740/bd014082-e14c-11e5-9907-2107467110e5.png)

**Comment by alexandru-m-g** [comment]
@cjhendrix What if 2 different titles generate the same "name" property ( and so the same URL). Then the message is not entirely correct.

**Comment by cjhendrix** [comment]
No, it would be a white lie, but the user wouldn't know it.  I don't think that matters.  Am I missing something?

**Comment by alexandru-m-g** [comment]
No, you are mostly right. The small issue here is that these messages are the same as the ones that users of the API get. So we probably need to tweak a bit the logic to change the message if it is rendered on the UI.
